lib/Text/PRN/Slurp.pm

```diff
@@ -130,6 +129,7 @@ sub _from_io_handler {
             }
         }
         else {
+            chomp $row;
```

Comment:
Since I don't know what is the correct/expected behaviour, I assumed the test is the expected behaviour and I changed the code to make the test pass, simple (Test Driven Coding).

If I know what we expect as the correct output then only I can suggest something :-)